Auxillary_files/MIP_grids.json

```diff
@@ -1,161 +1,173 @@
 {
-  "Header": {
-    "Conventions": "CF-1.7 CMIP-6.3",
-    "checksum": "",
-    "cmor_version": "4.0",
-    "data_specs_version": "6.3.0.0",
-    "generic_levels": "",
-    "int_missing_value": "-999",
-    "missing_value": "1e20",
-    "product": "model-output",
-    "table_date": "2023-11-16",
-    "table_id": "grids"
-  },
-  "axis_entry": {
-    "grid_latitude": {
-      "axis": "Y",
-      "long_name": "latitude in rotated pole grid",
-      "out_name": "rlat",
-      "standard_name": "grid_latitude",
-      "type": "double",
-      "units": "degrees"
+    "Header": {
+        "Conventions": "CF-1.7 CMIP-6.3",
+        "checksum": "",
+        "cmor_version": "4.0",
```

Comment:
This will likely prevent CMOR 3.7.3 from running - I believe it checks this prior to running, @matthew-mizielinski can confirm with his test setup

Reply:
We can delete almost the entire header here.  @durack1, these tables won't work with CMOR until you get a new version out.